frontend/src/store.js

```diff
@@ -41,6 +41,8 @@ const UIStore = new Store({
   meaOptions: [],
   organisationType: sectorOptions,
   sectorOptions: sectorOptions,
+  summary: [],
+  stakeholders: [],
```

Comment:
stakeholders api respose is not a vector but an object with following keys
`{stakeholders: [], page: x, limit: y, pages: z, count: w}`

Reply:
anyway, not sure if we need to initialise the values in this store or just assigning null is enough

Reply:
change the value to null